src/Documents/DocumentStoreBase.ts

```diff
@@ -14,7 +14,7 @@ import {
     BeforeConversionToEntityEventArgs,
     AfterConversionToEntityEventArgs,
     FailedRequestEventArgs,
-    TopologyUpdatedEventArgs
+    TopologyUpdatedEventArgs, BeforeRequestEventArgs, SucceedRequestEventArgs

```

Comment:
all other classes are in a new line. Can you do the same for those 2 remaining ones?

Reply:
done